presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorColumnIdentity.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor;
+
+import com.facebook.presto.spi.ColumnIdentity;
+import com.facebook.presto.spi.PrestoException;
+import com.google.common.primitives.Longs;
+
+import static com.facebook.presto.spi.StandardErrorCode.CORRUPT_SERIALIZED_IDENTITY;
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.io.BaseEncoding.base16;
+import static java.lang.String.format;
+import static java.util.Arrays.copyOfRange;
+import static java.util.Objects.hash;
+
+public final class RaptorColumnIdentity
+        implements ColumnIdentity
+{
+    private static final byte CURRENT_VERSION = 1;
+    private static final int SERIALIZED_SIZE = Byte.BYTES + Long.BYTES;
+
+    private final long columnId;
+
+    public RaptorColumnIdentity(long columnId)
+    {
+        this.columnId = columnId;
+    }
+
+    public static RaptorColumnIdentity deserialize(byte[] bytes)
+    {
+        checkArgument(bytes.length >= Byte.BYTES, "bytes for RaptorColumnIdentity is empty");
```

Comment:
Make the message "is corrupt" instead of "is empty", to avoid the confusion about `>= 1`